From d5c1a7811bf76abf22045f8f2f5c24fc0d1f6747 Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Sun, 12 Sep 2021 06:12:33 +0200
Subject: [PATCH 2/4] seatd-launch: respect PATH when looking for command

$ seatd-launch sway -c /dev/null
Could not start target: No such file or directory
---
 seatd-launch/seatd-launch.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/seatd-launch/seatd-launch.c b/seatd-launch/seatd-launch.c
index 0903fe1..c362a15 100644
--- a/seatd-launch/seatd-launch.c
+++ b/seatd-launch/seatd-launch.c
@@ -139,7 +139,7 @@ int main(int argc, char *argv[]) {
 		goto error_seatd;
 	} else if (child == 0) {
 		setenv("SEATD_SOCK", sockpath, 1);
-		execv(argv[1], &argv[1]);
+		execvp(argv[1], &argv[1]);
 		perror("Could not start target");
 		_exit(1);
 	}
-- 
2.33.0

